.github/workflows/update-ami.py

```diff
@@ -1,14 +1,14 @@
 #!/usr/bin/env python3
 import json
-import os 
+import os
 import boto3
 import uuid
 import re
 import subprocess
 
 # Globals
 amifile = '.github/workflows/amichange.json'
-packerfile = ".github/packer/ubuntu-jammy-x86_64-public-ami.json"
+packerfile = ".github/packer/ubuntu-jammy-x86_64-public-ami.json.pkr.hcl"
```

Comment:
comment for line 18:

remove capital `t`
skip_create_ami = os.getenv('SKIP_CREATE_AMI', "true")